connector/connect/server/src/main/scala/org/apache/spark/sql/connect/planner/SparkConnectPlanner.scala

```diff
@@ -2466,14 +2466,16 @@ class SparkConnectPlanner(val session: SparkSession) {
 
       case StreamingQueryManagerCommand.CommandCase.GET =>
         val query = session.streams.get(command.getGet)
```

Comment:
'getGet` sounds off. May be we could rename to `get_query` (changing the name is ok, not an incompatible change at this stage).